packages/main/src/MultiComboBox.js

```diff
@@ -0,0 +1,403 @@
+		_inputLiveChange: { type: Function },
+		_inputChange: { type: Function },
+		_filteredItems: { type: Object },
+		_iconPressed: { type: Boolean },
+	},
+	events: /** @lends sap.ui.webcomponents.main.MultiComboBox.prototype */ {
+		/**
+		 * Fired when the input operation has finished by pressing Enter or on focusout.
+		 *
+		 * @event
+		 * @public
+		 */
+		change: {},
+
+		/**
+		 * Fired when the value of the <code>ui5-multi-combobox</code> changes at each keystroke.
+		 *
+		 * @event
+		 * @public
+		 */
+		input: {},
+
+		/**
+		 * Fired when selection is changed by user interaction
+		 * in <code>SingleSelect</code> and <code>MultiSelect</code> modes.
+		 *
+		 * @event
+		 * @param {Array} items an array of the selected items.
+		 * @public
+		 */
+		selectionChange: {
+			detail: {
+				items: { type: Array },
+			},
+		},
+	},
+};
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ *
+ * An entry posted on the timeline.
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webcomponents.main.MultiComboBox
+ * @extends UI5Element
+ * @tagname ui5-timeline
+ * @public
+ */
+class MultiComboBox extends UI5Element {
+	static get metadata() {
+		return metadata;
+	}
+
+	static get renderer() {
+		return MultiComboBoxRenderer;
+	}
+
+	static get calculateTemplateContext() {
+		return state => {
+			return {
+				ctr: state,
+				editable: !state.readonly,
+				selectedItemsListMode: state.readonly ? "None" : "MultiSelect",
+				classes: {
+					icon: {
+						[`ui5-multi-combobox-icon-pressed`]: state._iconPressed,
+						[`ui5-multi-combobox--icon`]: true,
+					},
+				},
+			};
+		};
+	}
+
+	static get styles() {
+		return styles;
+	}
+
+	constructor() {
+		super();
+
+		this._filteredItems = [];
+		this._inputLastValue = "";
+		this._deleting = false;
+
+		this._showMorePopover = event => {
+			this._togglePopover(true);
+		};
+
+		this._showAllItemsPopover = event => {
+			this._togglePopover(false);
+		};
+
+		this._allItemsSelectionChange = event => {
+			this._listSelectionChange(event);
+		};
+
+		this._selectedItemsSelectionChange = event => {
+			this._listSelectionChange(event);
+		};
+
+		this._inputLiveChange = this._handleInputLiveChange.bind(this);
+		this._tokenDelete = this._handleTokenDelete.bind(this);
+		this._tokenizerFocusOut = this._handleTokenizerFocusOut.bind(this);
+
+		this._inputChange = () => this.fireEvent("change");
+
+		this._afterAllPopoverClose = () => {
+			this._toggleIcon();
+		};
+
+		this._afterAllPopoverOpen = () => {
+			this._toggleIcon();
+		};
+
+		this._keydown = this._handleKeyDown.bind(this);
+	}
+
+	_handleInputLiveChange(event) {
+		const input = event.target;
+		const value = input.value;
+		const filteredItems = this._filterItems(value);
+		const oldValueState = input.valueState;
+
+		if (!filteredItems.length && value && this.validateInput) {
+			input.value = this._inputLastValue;
+			input.valueState = "Error";
+
+			setTimeout(() => {
```

Comment:
This seems like a workaround, maybe there is something that lacks in the framework.
I think we should at least talk about it briefly, if we can do something else.